src/shared/sleep-config.h

```diff
@@ -23,6 +23,17 @@ typedef struct SleepConfig {
 void free_sleep_config(SleepConfig *sc);
 DEFINE_TRIVIAL_CLEANUP_FUNC(SleepConfig*, free_sleep_config);
 
+/* entry in /proc/swaps */
+typedef struct SwapEntry {
+        char *device;
+        char *type;
+        size_t size;
+        size_t used;
```

Comment:
hmm, i figure this should be uint64_t, not size_t. After all we might be 32bit userspace on a 64bit kernel (i understand the old code didn't do this properly either, but I figure while this is touched we should fix that, too)